app/views/podcasts/show.rss.builder

```diff
@@ -110,6 +110,8 @@ xml.rss "xmlns:atom" => "http://www.w3.org/2005/Atom",
       xml.podcast :follow, url: @podcast.subscribe_links_path
     end
 
+    xml.podcast :locked, @feed.import_locked ? "yes" : "no"
```

Comment:
Thinking of cases where we'd lock some Feeds, but not others.

Maybe we'd make a new private RSS that had both public-RSS and bonus episodes, let the new host import that, but keep the real feeds locked that entire time?

Or can you think of others?

Reply:
@brandonhundt I put this `podcast:locked` checkbox at the feed level, not the podcast level?
As @cavis is pointing out, that's perhaps unnecessary, as when I go to move a feed I might move all the feeds?

I was also thinking in the process of a feed transition, we might create an ad-free private feed that is unlocked, while keeping the main feed locked, so it made sense to have at the feed level for that common transition case?

Reply:
(I actually think it's a good idea to put it at the Feed level... just thinking through cases for it).